app/controllers/site_controller.rb

```diff
@@ -714,13 +758,22 @@ def set_study_download_options
     end
   end
 
+  def set_reviewer_access
+    @reviewer_access = ReviewerAccess.find_by(access_code: params[:access_code])
+    unless @reviewer_access.present?
+      redirect_to merge_default_redirect_params(site_path, scpbr: params[:scpbr]),
+                  alert: 'Invalid access code; please check the link and try again.' and return
```

Comment:
yeah, I notice we're keeping the `and return` here, which I like

Reply:
See the comment about `head` - it is necessary here, but not with `head`.